### ISSUE TITLE
F_fdisk returned 19!

### ISSUE THREAD

**Issue by Fane447** [suchmememanyskill/TegraExplorer#8]
I tried using version 1.2 but trying to format the SD card with either option just instantly displayed 'f_fdisk returned 19!' after the partition information.

Using version 1.1.1 worked fine.

(It was a 128gb SD card, let me know if you need anymore information.)

**Comment by suchmememanyskill** [comment]
Can i get the sector counts for what tegraexplorer is trying to format the disk as?

**Comment by NateDogg1232** [comment]
I am getting the same thing. I have a 64gb card. I also get it when I try to just format the card as FAT32.

```
TotalSectors: 125042688
Partition1 (SD): 63897600
Partition2 (EMUMMC): 61145088
```

I will use the Version 1.1.1 for now, and I won't be able to help further, as I just want to get this working, as right now I have SysNAND CFW, but I hope this helps!

**Comment by suchmememanyskill** [comment]
Issue has been fixed locally. Will do some more testing to make sure it works

**Comment by suchmememanyskill** [comment]
fixed in commit 1410fa1
